chart/values.schema.json

```diff
@@ -3047,12 +3047,12 @@
           "$ref": "#/$defs/SyncRewriteHosts",
           "description": "RewriteHosts is a special option needed to rewrite statefulset containers to allow the correct FQDN. virtual cluster will add\na small container to each stateful set pod that will initially rewrite the /etc/hosts file to match the FQDN expected by\nthe virtual cluster."
         },
-        "translate": {
+        "patches": {
           "items": {
             "$ref": "#/$defs/TranslatePatch"
           },
           "type": "array",
-          "description": "Translate the patch according to the given patches."
+          "description": "Patches the resource according to the given patches."
```

Comment:
This is ambiguous, maybe keep "translate" here?

Reply:
Patch is a common word used to do exactly what we're doing which is update/change a resource. 

https://kubernetes.io/docs/reference/kubectl/generated/kubectl_patch/